docs/manifest.md

```diff
@@ -0,0 +1,221 @@
+that functionality (e.g. `slides` is forbidden, but `sliding` or `slitastic`
+would be ok). We have this rule because users will find packages with these
+canonical names first, creating an unfair advantage for the package author who
+claimed that name. Names should not include the word "typst" (as it is
+redundant). If they contain multiple words, names should use `kebab-case`. Look
+at existing packages and PRs to get a feel for what's allowed and what's not.
+
+*Additional guidance for template packages:* It is often desirable for template
+names to feature the name of the organization or publication the template is
+intended for. However, it is still important to us to accommodate multiple
+templates for the same purpose. Hence, template names shall consist of a unique,
+non-descriptive part followed by a descriptive part. For example, a template
+package for the fictitious _American Journal of Proceedings (AJP)_ could be
+called `organized-ajp` or `eternal-ajp`. Package names should be short and use
+the official entity abbreviation. Template authors are encouraged to add the
+full name of the affiliated entity as a keyword.
+
+The unamended entity name (e.g. `ajp`) is reserved for official template
+packages by their respective entities. Please make it clear in your PR if you
+are submitting an official package. We will then outline steps to authenticate
+you as a member of the affiliated organization.
+
+If you are an author of an original template not affiliated with any
+organization, only the standard package naming guidelines apply to you.
+
+These rules also apply to names in other languages, including transliterations
+for languages that are not generally written using the latin alphabet.
+
+### Writing a good description
+
+A good package description is simple, easily-understandable and succinct. Here
+are some rules to follow to write great descriptions:
+
+- Keep it short. Try to maximise the content to length ratio and weigh your words
+  thoughtfully. Ideally, it should be 40 to 60 characters long.
+
+- Terminate your description with a full stop.
+
+- Avoid the word "Typst", which is redundant unless your package or template
+  actually has to do with Typst itself or its ecosystem (like in the case of
+  [mantys] or [t4t]).
+
+- Avoid the words "package" for packages and "template" for templates; instead:
+  - Packages allow the user to *do* things; use the imperative mood. For
+    example, `Draw Venn diagrams.` is better than `A package for drawing Venn
+    diagrams`.
+  - Templates allow the user to write certain *types* of documents; clearly
+    indicate the type of document your template allows. For example, `Master’s
+    thesis at the Unseen University.` is better than `A template for writing a
+    master’s thesis at the Unseen University.`. Omit the indefinite article ("A
+    …", "An …") at the beginning of the description for conciseness.
```

Comment:
From these examples, it's unclear whether descriptions should end with a full stop.